validation/raw_values.rst

```diff
@@ -51,6 +51,23 @@ Validation of arrays is possible using the ``Collection`` constraint::
 
     $validator = Validation::createValidator();
 
+    $input = array(
+        'name' => array(
+          'first_name' => 'Fabien',
+          'last_name' => 'Potencier',
+        ),
+        'email' => 'test@email.tld',
+        'simple' => 'hello',
+        'gender' => 3,
+        'file' => null,
+        'password' => 'test',
+        'tags' => array(
+            array('slug' => 'symfony_doc', 'label' => 'symfony doc'),
```

Comment:
I would make it multi-lines as well

Reply:
Ok
Did you know if we could use the short syntax for array instead?